api/src/test/java/org/pmiops/workbench/workspaces/WorkspaceServiceTest.java

```diff
@@ -159,7 +183,30 @@ public void activeStatus() {
     EnumSet.allOf(WorkspaceActiveStatus.class)
         .forEach(
             status ->
-                assertThat(mockDbWorkspace("1", "1", status).getWorkspaceActiveStatusEnum())
+                assertThat(mockDbWorkspace(workspaceIdIncrementer.getAndIncrement(),"1", status).getWorkspaceActiveStatusEnum())
                     .isEqualTo(status));
   }
+
+  @Test
+  public void getRecentWorkspaces() {
+    for (int i = 1; i < WorkspaceServiceImpl.RECENT_WORKSPACE_COUNT; i++) {
+      org.pmiops.workbench.db.model.Workspace workspace = workspaces.get(i);
+      workspaceService.updateRecentWorkspaces(workspace.getWorkspaceId(), USER_ID, Timestamp.from(NOW));
+    }
+    List<UserRecentWorkspace> recentWorkspaces = workspaceService.getRecentWorkspaces(USER_ID);
+    for (int i = 0; i < recentWorkspaces.size(); i++) {
```

Comment:
You can do this without a loop using `assertThat(recentWorkspaces).containsAll(workspaces)` or something similar.